src/runtime/tests/artifacts/Words/source/GamePane.js

```diff
@@ -11,7 +11,7 @@
 
 /* global defineParticle */
 
-defineParticle(({SimpleParticle, html, log, resolver}) => {
+defineParticle(({DomParticle, html, log, resolver}) => {
```

Comment:
why unlike all other particles here you replace SimpleParticle with DomParticle instead?

Reply:
The entire point of having a separate corpus of tests in `tests/artifacts` was to insulate them from changes in the main set of Particles.

Often the tests that use these files is not testing DomParticle per se, it's just using this infrastructure to test other things. 

From my perspective, changing anything in this test corpus without deliberation is asking for trouble.